float64 axisX
float64 axisY
float64 axisZ

float64 axisRX
float64 axisRY
float64 axisRZ

int32 hatX
int32 hatY

bool[11] buttons
